index.html

```diff
@@ -1192,33 +1192,31 @@ <h3>
       </p>
 
       <p>
-        A DID method specification should have a section dedicated to personal
-        data covering the extent to
+        DID method specifications are encouraged to have a section dedicated to
+        personal data, covering the extent to
```

Comment:
```suggestion
        DID method specifiers are encouraged to include a specification section
        dedicated to personal data, covering the extent to
```

Reply:
I think they are actually required to do this, by DID Core.... Privacy and Security considerations are requirements of a did method spec... maybe we can say some thing like "try and make these really impressive"... but still some folks will probably think they didn't go far enough.

Reply:
Committed suggested change in b5dc79571ed8382216870c9bcfe2f035a272c1a5.

---

I don't see "personal data" mentioned in these requirements:
- https://www.w3.org/TR/did-core/#security-requirements "requirements for all DID method specifications when authoring the Security Considerations section"
- https://www.w3.org/TR/did-core/#privacy-requirements "requirements for all DID method specifications when authoring the Privacy Considerations section"

But it is discussed in non-normative sections:

- https://www.w3.org/TR/did-core/#binding-to-physical-identity "A DID and DID document do not inherently carry any personal data and it is strongly advised that non-public entities do not publish personal data in DID documents."
- https://www.w3.org/TR/did-core/#encrypted-data-in-did-documents "placing encrypted data in a DID document is not an appropriate means to protect personal data."
- https://www.w3.org/TR/did-core/#keep-personal-data-private "If a DID method specification is written for a public-facing verifiable data registry where corresponding DIDs and DID documents might be made publicly available, it is critical that those DID documents contain no personal data." ...

We could say that the personal data section is encouraged in addition to the Privacy and Security considerations sections which are required. Or would personal data be a subsection of Privacy and/or Security considerations?

I agree that more encouragement to develop these sections would be good.